pageserver/src/tenant_config.rs

```diff
@@ -220,29 +219,4 @@ impl TenantConf {
             trace_read_requests: false,
         }
     }
-
-    pub fn dummy_conf() -> Self {
-        TenantConf {
-            checkpoint_distance: defaults::DEFAULT_CHECKPOINT_DISTANCE,
-            checkpoint_timeout: Duration::from_secs(600),
-            compaction_target_size: 4 * 1024 * 1024,
-            compaction_period: Duration::from_secs(10),
-            compaction_threshold: defaults::DEFAULT_COMPACTION_THRESHOLD,
-            gc_horizon: defaults::DEFAULT_GC_HORIZON,
-            gc_period: Duration::from_secs(10),
-            image_creation_threshold: defaults::DEFAULT_IMAGE_CREATION_THRESHOLD,
-            pitr_interval: Duration::from_secs(60 * 60),
```

Comment:
![изображение](https://user-images.githubusercontent.com/3599598/209129622-1f032e01-f493-44ce-a9a0-a42e1e49eec1.png)

But seriously, `pitr_interval` used to be 60 mins, and now it's `DEFAULT_PITR_INTERVAL`, which is

```rust
pub const DEFAULT_PITR_INTERVAL: &str = "30 days";
```

right?

Reply:
These values were not considered (by me, at least) for a while.
I've removed the ones used in unit tests and the default ones are now used everywhere, so technically I've not changed anything with this PR, since the tests are green.

Yet, I agree that we should adjust those.

Reply:
Cool, I just wanted to double-check! :D